src/pages/index.jsx

```diff
@@ -2,7 +2,152 @@ import React from "react";
 import Layout from "src/layout";
 
 const Index = () => {
-  return <Layout>index</Layout>;
+  return (
+    <Layout>
+      Arcu cursus euismod quis viverra nibh? Pharetra diam sit amet nisl
```

Comment:
is there a way to generate this through a `.md` file instead of hardcoding it? (even if it is the homepage)